Source/Internal/Table/PDFTableObject.swift

```diff
@@ -303,9 +304,12 @@ class PDFTableObject: PDFRenderObject {
                     cellFrame.origin.y += table.margin
                     contentFrame.origin.y -= startPosition.y - minOffset
                     contentFrame.origin.y += table.margin
-
-                    pageStart = pageStart == .null ? cellFrame.origin : pageStart
-                    pageEnd = CGPoint(x: cellFrame.maxX, y: cellFrame.maxY) + CGPoint(x: table.margin, y: table.margin)
+                    
+                    // Maintain pageStart and pageEnd
+                    pageStart.x = min(cellFrame.minX, pageStart.x)
```

Comment:
Do we have documentation available on how the min/max functions work with `CGPoint.null`?

Reply:
`.null` is defined in this codebase as `.infinity` so anything will be less. That's why I had to redefine `pageEnd` to `CGPoint()` aka `0,0` so `max` works as desired